Assets/PatchKit Patcher/Scripts/AppUpdater/AppUpdater.cs

```diff
@@ -65,6 +65,12 @@ public void Update(CancellationToken cancellationToken)
         private void HandleFallback( CancellationToken cancellationToken)
         {
             StrategyType fallbackType = _strategyResolver.GetFallbackStrategy( _strategy.GetStrategyType() );
+
+            if ( fallbackType == StrategyType.None )
+            {
+                throw new Exception("Updater Strategy not found");
```

Comment:
I'd prefer to change the return type of `HandleFallback` to `bool` and return `false` in case of no possibility of fallback to use it to rethrow the exception raised by operation that caused the attempt for fallback.

Reply:
Done in latest commit